libmcount/dynamic.c

```diff
@@ -73,6 +74,26 @@ static struct mcount_orig_insn *lookup_code(struct rb_root *root,
 	return iter;
 }
 
+static inline bool frozen(struct code_page *cp)
+{
+	return cp->frozen;
+}
+
+static struct code_page *alloc_codepage()
```

Comment:
Please add `void` in the argument, it's a standard in C.

Reply:
thanks for advice, I'll change it as follow as your comment.